tools/src/process-allure-reports.py

```diff
@@ -0,0 +1,101 @@
+import os
+import subprocess
+import argparse
+from allure_combine import combine_allure
+
+COMBINE_DIR = 'combine'
+COMPLETE_FILE_NAME = 'index.html'
```

Comment:
minor: i suggest to clarify the meaning a bit
```suggestion
COMPLETE_FILE_NAME = 'index.html' # file to write COMBINE_DIR/complete.html file data into
```